src/prpy/planning/tsr.py

```diff
@@ -85,7 +86,7 @@ def PlanToIK(self, robot, goal_pose, **kw_args):
         return self.PlanToTSR(robot, [tsr_chain], **kw_args)
     """
```

Comment:
It's not related to the pull request, but we really should delete this dead code.